test/Turn-test.js

```diff
@@ -0,0 +1,46 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const Turn = require('../src/Turn');
+const Card = require('../src/Card')
+
+describe('Turn', function() {
+
+  it('should be a function', function() {
+    const turn = new Turn();
+    expect(Turn).to.be.a('function');
+  });
+
+  it('should be an instance of a Turn', function() {
+    const turn = new Turn();
+    expect(Turn).to.be.an.instanceof(Turn);
+  });
+
+  it('should store a user guess to the question', function() {
+    const turn = new Turn('my guess');
+    expect(Turn.guess).to.equal('my guess');
+  });
+
+  it('should store a Card object for the current card in play', function() {
+    const card = new Card()
+    const turn = new Turn('my guess', card)
+    expect(Turn.card).to.equal(card)
```

Comment:
The one I'm most curious/worried about. Excited to write the class and see if this works.